lib/propre.rb

```diff
@@ -39,11 +39,9 @@ def self.remove_patterns(arg)
   end
 
   def self.sanitize(arg)
-    if arg.count('.') > arg.count('-')
-      arg.gsub('.', ' ').strip
-    else
-      arg.gsub('-', ' ').strip
-    end
+    separator = %w(- _ .).each_with_object({}) { |c, s| s[c] = arg.count(c) }
+      .sort_by(&:last).reverse.first.first
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.